templates/nav.html

```diff
@@ -60,15 +58,15 @@
           <a class="nav-link text-light" href="/bydate.html?datafilename=raw">Chart/All</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link text-light" target="_blank" href="https://datastudio.google.com/s/n2u7zjB6voE">View in Data Studio</a>
+          <a class="nav-link text-light" href="/report_tree.html">Static Reports</a>
```

Comment:
Did you mean to change this line?
I think this was a recent change from @markroth8 ?

Reply:
no, I didn't.  that is a merge conflict :-(

Reply:
And more generally are there any changes to `nav.html` that are intentionally part of this PR?

Reply:
no.  no actual changes intended.

Reply:
looks like the only changes are formatting changes at this point